Backend/getPassengersOnRoute.php

```diff
@@ -0,0 +1,44 @@
+<?php
+require "init.php";
+
+$user_email= "anosirrah365@gmail.com";//$_POST["loggedInUserEmail"];
+$routeName= "From Deez To Nutz";//$_POST["routeName"];
+
+//get route id
+$sql = "SELECT route_id FROM Routes_Users_Association WHERE user_id  = (SELECT id FROM Users WHERE email = '".$user_email."')
+            AND userRouteName = '".$routeName."'";
+    
+$result = mysqli_query($con,$sql);
+
+
+if(mysqli_num_rows($result) == 0){
```

Comment:
Control statement "if" should have one space between the control keyword and opening parenthesis according to PHP conventions.